scripts/util.py

```diff
@@ -383,16 +448,8 @@ def compute_service(credentials=None, user_agent=USER_AGENT, version="beta"):
     """Make thread-safe compute service handle
```

Comment:
We never pass `credentials` here , let's remove it and simplify logic of `get_credentials`

Reply:
I think I have addressed this, PTAL and let me know.

Reply:
please remove argument `credentials` since it's not used.